x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/diagnostics/index_templates.spec.ts

```diff
@@ -19,27 +18,7 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
   const start = new Date('2021-01-01T00:00:00.000Z').getTime();
   const end = new Date('2021-01-01T00:15:00.000Z').getTime() - 1;
 
-  describe.skip('Diagnostics: Index Templates', () => {
-    describe('When there is no data', () => {
```

Comment:
When default index templates are deleted, they get recreated automatically. Introduced here: https://github.com/elastic/elasticsearch/pull/97546

This means we can't perform check against empty response anymore.